_components/search-bar.md

```diff
@@ -9,11 +9,13 @@ lead: Intro text on what is included in this section and how to use it. No more
   <div class="usa-grid">
     <div class="usa-width-one-half">
       <form class="usa-search usa-search-big">
-        <label for="search-field-big">Search Big</label>
-        <input type="search" id="search-field-big">
-        <button type="submit">
-          <span class="usa-search-submit-text">Search</span>
-        </button>
+        <div role="search">
```

Comment:
Why was adding this div done here? Accessibility? I remember Bristow saying this was fine as is.